render_files.py

```diff
@@ -175,6 +191,7 @@ def main():
     install_config = os.environ.get("INSTALLER_CONFIG")
     cluster_id = os.environ.get("CLUSTER_ID")
     inventory_endpoint = os.environ.get("INVENTORY_ENDPOINT")
+    dhcp_allocation_file = os.environ.get("DHCP_ALLOCATION_FILE")
```

Comment:
how does it related to BMH? and I would call DHCP_ALLOCATION_FILE diffrently @ori-amizur

Reply:
It is not.  Why do you think it is?

Reply:
because it's in the same function?